compiler/hie-core/src/Development/IDE/Core/Compile.hs

```diff
@@ -202,13 +202,14 @@ newtype WriteInterface = WriteInterface Bool
 mkTcModuleResult
     :: GhcMonad m
     => WriteInterface
```

Comment:
Should we make this a `Maybe FilePath` instead of having a `Bool` and a `FilePath`?

Reply:
Changed